File: matematica_computacional/matematica.py

""" FUNÇÃO """
class funcao:

    def __init__(self, x, y):
        self.x = x
        self.y = y

    def funcao_primeiro_grau(self):
        y = (2 * self.x) + 1
        return y

""" FUNÇÃO """


a = funcao(-2,0)
print(a.funcao_primeiro_grau())

File: matematica_computacional/matematica.js
export class Funcao{
    
    constructor(x, y, z){
        this.x = x;
        this.y = y;
        this.z = z;
    }

    funcao_primeiro_grau(){
        return this.y = (this.z * this.x) + 1
    }

    calc_funcao() {
    
        var arr = new Array();
    
        while (this.x < this.y) {
            let a = this.funcao_primeiro_grau();
            arr.push([this.x, a]);
            x++
        }
    
        return arr;
    
    }
}

/* export {Funcao}; */
